Sources/mpc-core-kit-swift/Helper.swift

```diff
@@ -41,13 +41,13 @@ func factorDescriptionToJsonStr ( dataObj: [String: Any]  ) throws -> String {
 }
 
 
-public func hashMessage(message: Data) -> String {
-    let hash = message.sha3(.keccak256)
+public func hashMessage(message: Data) throws -> String {
+    let hash = try message.sha3(varient: Variants.KECCAK256)
     return hash.base64EncodedString()
 }
 
-public func hashMessage(message: String) -> String {
-    return hashMessage(message: Data(message.utf8))
+public func hashMessage(message: String) throws -> String {
+    return try hashMessage(message: Data(message.utf8))
 }
```

Comment:
Since you're importing `tssClientSwift` here, you can use TssHelpers.hashMessage(message: String) and remove this since it does the same thing.

See:
https://github.com/torusresearch/tss-client-swift/blob/e2a7a3d0b4d2bedd5e74a7ba5e43d562040b3db6/Sources/tssClientSwift/Helpers.swift#L16

Is the sha3 import a new or existing dependency?

Reply:
existing

Reply:
Can it be removed by doing the above instead?